autonomy/cli/build_images.py

```diff
@@ -54,23 +54,17 @@ def build_image(
 ) -> None:
     """Build runtime images for autonomous agents."""
 
-    try:
+    with reraise_as_click_exception(FileNotFoundError):
         if agent is None:
             service_dir = Path(service_dir or Path.cwd()).absolute()
             service = load_service_config(service_dir)
             agent = service.agent
-    except FileNotFoundError as e:
-        raise click.ClickException(
-            "Service configuration not found the current directory"
-        ) from e
```

Comment:
this changes the error message here, had to adjust the check in the test. I don't think there is any loss of meaningful information whatsoever though.

Reply:
Suggests that method is lacking an optional argument for a custom error message